core/drivers/zynqmp_pm.c

```diff
@@ -0,0 +1,144 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (C) 2021 Foundries.io Ltd
+ */
+
+#include <arm.h>
+#include <drivers/zynqmp_pm.h>
+#include <kernel/cache_helpers.h>
+#include <mm/core_memprot.h>
+#include <smccc.h>
+#include <string.h>
+#include <tee/cache.h>
+#include <tee_api_types.h>
+#include <types_ext.h>
+#include <utee_defines.h>
+
+#define EFUSE_ACCESS_SMC	0xC2000035
```

Comment:
Do we have any documentation on this interface? A link to some reference manual would be nice.